contracts/0.8.9/SelfOwnedStETHBurner.sol

```diff
@@ -68,16 +74,27 @@ interface IOracle {
   * @dev Burning stETH means 'decrease total underlying shares amount to perform stETH token rebase'
   */
 contract SelfOwnedStETHBurner is IBeaconReportReceiver {
+    uint256 private constant MAX_BASE_POINTS = 10000;
+
     uint256 private coverSharesBurnRequested;
     uint256 private nonCoverSharesBurnRequested;
 
     uint256 private totalCoverSharesBurnt;
     uint256 private totalNonCoverSharesBurnt;
 
+    uint256 private maxBurnAmountPerRunBasePoints = 4; // 0.04% by default for the biggest `stETH:ETH` curve pool
+
```

Comment:
Please consider to set the variable via constructor. It will save one extra call for setting via setter on deploy. 🙌

Reply:
I'll do it, an easy one :+1: